Services/TokenAuthentication/ModsenOnlineStore.Login.API/Controllers/LoginController.cs

```diff
@@ -69,13 +71,44 @@ public async Task<IActionResult> RegisterUserAsync(AddUserDTO user)
             return Ok(response.Message);
         }
 
+        [HttpPost]
+        [Authorize(Roles = "User")]
+        [Route("/NewPayment/{userId}")]
+        public async Task<IActionResult> MakePaymentAsync(int userId, decimal money)
+        {
+            var response = await userMoneyService.MakePaymentAsync(userId, money);
+
+            if (!response.Success)
+            {
+                return NotFound(response);
+            }
+
+            return Ok(response);
+        }
+
+        [HttpPost]
+        [Authorize(Roles = "Admin")]
+        [Route("/AddMoney/{userId}")]
```

Comment:
the same

Reply:
endpoint route moved to [HttpPost] attribute